exercises/practice/leap/Leap.roc

```diff
@@ -0,0 +1,4 @@
+module [isLeapYear]
+
+isLeapYear = \year ->
+    year > 0
```

Comment:
Is there like an exception/error/todo construct that can be used here?

Reply:
Yes, you can use `crash "TODO"`.

Reply:
I would suggest using that in the stub files (the solution files the student starts out with)

Reply:
Ah yes, I wasn't sure what to do here. `crash "TODO"` sounds good, thanks.

Reply:
In my experience, a message like `"TODO"` doesn't always convey our intent that a student should add their own code to the stub file. `hello-world` provides code that they need to edit to pass the tests which is very different than what we're doing here. So a student might be expecting something similar when it comes time to solve `leap`. Then, it's unclear whether the TODO is for the track maintainers or the student or what is expected. Something like "crash "please implement this function"` is clear on what needs to be done. If you go that route, it might be helpful to specify the name of the function as well. If there are multiple functions being tested, it'd look like `crash "please implement functionA"` and `crash "please implement functionB"` and that'd be potentially helpful context in the results window.